benchmarking/grade_output.py

```diff
@@ -1,39 +1,269 @@
-from helpers.classes import GeneratedProblemStatement, MinerOutputScore, IssueSolution
+from sweagent.environment.swe_env import EnvironmentArguments, SWEEnv
+from sweagent.agent.agents import AgentArguments
+from sweagent.environment.swe_env import EnvironmentArguments
+from sweagent.agent.models import ModelArguments
+
+from typing import Dict, List
+import pytest
+from pathlib import Path
+import json
+import tempfile
+
+from helpers.classes import GeneratedProblemStatement, TestResults, MinerSolutionTestResults, IssueSolution, MinerSolutionScore, MinerLLMEvaluation, ScriptArguments, ActionsArguments
 from helpers.clients import OPENAI_CLIENT
-import statistics
 
-def compute_overall_score(miner_output_score: MinerOutputScore) -> float:
-    DYNAMIC_CHECKLIST_WEIGHT = 0.2
-    ADDRESSES_PROBLEM_WEIGHT = 0.3
-    LOGICAL_SOLUTION_WEIGHT = 0.25
-    BREVITY_WEIGHT = 0.05
-    POTENTIAL_BUGS_WEIGHT = 0.2
+TEST_PATHS_BY_REPO = {
+    "seaborn": {
+        "path": "tests",
+        "label_side": "left",
+        "framework": "pytest"
+    }
+}
+
+SUPPORTED_TEST_FRAMEWORKS = ["pytest"]
+
+def create_script_arguments(model_name: str, repo_path: Path) -> ScriptArguments:
```

Comment:
Can we just reuse the same `create_script_arguments` func from `generate_solution.py`?